ansible/roles/tor_bridge/hosts

```diff
@@ -0,0 +1,7 @@
+[bridges]
+37.218.240.79 tor_nickname=Joubini obfs4_listen_port=8080
+1.209.122.142 tor_nickname=Hapalochlaena obfs4_listen_port=5222
+199.119.112.170 tor_nickname=Bimaculoides obfs4_listen_port=5223
```

Comment:
`37.218.240.79` and `199.119.112.170` seem to be still alive. Should they be included into `inventory` somehow? Are they _part of OONI infrastructure_?
`1.209.122.142` seems to be dead.

Reply:
`199.119.112.170` is now dead. `37.218.240.79` is not in the ooni eclipsis account so I think we can assume these as irrelevant.
@anadahz do you agree? If so I suggest we delete this branch.

Reply:
Please go ahead and delete this branch.
For reference two these bridges are still online (one on a different IP).

Does OONI has any obfs4 bridges running at the moment?

Reply:
> Does OONI has any obfs4 bridges running at the moment

Nope, only tor nodes in some networks to act as test-helpers during response to some tor-blocking incident.

I currently see no point to run `obfs4` bridges as part of OONI infra as IP:Port based blocking can be debugged with Tor relay as well. And obfs4-specific protocol-based blocking does not need a bridge that was online "long enough" to get blocked.